src/parser/shared/modules/ResourceGraph.tsx

```diff
@@ -8,12 +8,26 @@ abstract class ResourceGraph extends Analyzer {
   /** Implementer must override this to return the ResourceTracker for the resource to graph */
   abstract tracker(): ResourceTracker;
 
+  /** Implementer may set this to include the wasted line on the graph.
+   *  Note that the ResourceTracker must have events with `Event.changeWaste` for this to work.
+   */
+  includeWasted: boolean = false;
+
   /** Implementer may override this to give the graph line a custom color.
    *  Color must be in format '#rrggbb', where rr gg and bb are hex values. */
   lineColor(): string | undefined {
     return undefined;
   }
 
+  /** Implementer may override this to give the wasted line a custom color. */
+  wastedColor(): string {
```

Comment:
Should be a template literal instead of simply a string to aid implementers? 
Something like 
```
`rgba(${number},${number},${number},${number})`
```

Reply:
I could see wanting to support hex colors as an option too.

Reply:
If the lib supports it just add it as a union? Dunno what is supported by whatever takes in the opt

Reply:
I see that vega has a "Color" type, but which basically just are strings with aliases. I did some digging, but it seems like a proper hex type completely blows up the TS type checker. 

I've extended the docstring and changed `string` to `Color` instead, which hopefully should inform users of what is expected.